cypress/cypress.config.js

```diff
@@ -1,5 +1,10 @@
 module.exports = {
   projectId: 'htzagd',
+  env: {
```

Comment:
just out of curiosity, what is this doing?

Reply:
Legacy. (I forgot to delete this) Previously, I was using a modified version of cypress-react-selector to query to react components. But since that proved to be not working, I took out the code. This env variable was used to find the root of React DOM.